dbsnp-importer/src/main/java/uk/ac/ebi/eva/dbsnpimporter/jobs/steps/processors/SubSnpCoreFieldsToVariantProcessor.java

```diff
@@ -135,4 +156,27 @@ private int getAlleleIndex(String allele, String ref, String alt, String[] secon
     private Map<String, String> getSampleDataFromGenotypeCode(String genotypeCode) {
         return Collections.singletonMap("GT", genotypeCode);
     }
+
+    private VariantSourceEntry addFrequenciesToVariantSourceEntry(SubSnpCoreFields subSnpCoreFields, Variant variant,
+                                                                  VariantSourceEntry variantSourceEntry) throws IOException {
+        if (!areFrequenciesInvalid(subSnpCoreFields)) {
+            // TODO: do we need setFormat("AF")?
```

Comment:
@jmmut what do you think?
 Maybe setFormat("AF"), setFormat("GT") and setFormat("AF:GT") depending if there are frequencies, genotypes, or both?

Reply:
"AF" is added to `files.attrs`, which is the database kind of equivalent to the INFO column in a VCF file. It has nothing to do with the FORMAT column.